changelogs/fragments/6819-redfish-utils-add-links-parameter-for-get_disk_inventory.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - redfish_utils - support ``Volumes`` in response for ``GetDiskInventory`` (https://github.com/ansible-collections/community.general/pull/6819).
```

It might make sense to (also) mention which modules this affects. Users cannot invoke module utils directly.
```suggestion
  - redfish_utils module utils - support ``Volumes`` in response for ``GetDiskInventory`` (https://github.com/ansible-collections/community.general/pull/6819).
```